addons/dragon/data/M47mis.rvmat

```diff
@@ -0,0 +1,81 @@
+class StageTI {
+    texture="z\ace\addons\dragon\data\M47_TI_CA.paa";
+};
+ambient[]={0.989,1,0.958,1};
+diffuse[]={0.989,1,0.958,1};
+forcedDiffuse[]={0,0,0,0};
+emmisive[]={0,0,0,1};
+specular[]={0.159,0.212,0.196,1};
+specularPower=90;
+PixelShaderID="Super";
+VertexShaderID="Super";
+class Stage1 {
+    texture="z\ace\addons\dragon\data\M47mis_nohq.paa";
```

Comment:
File doesn't exist. Aka name is not correct when read case-sensitively.
Armake reads stuff case sensitive though.
Make sure that the references inside the p3d are also correct when handles case sensitively.